mzPlot/Plots/BarPlot.cs

```diff
@@ -0,0 +1,27 @@
+﻿using BayesianEstimation;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using MzLibUtil;
+using OxyPlot;
+using System.Linq;
+using OxyPlot.Series;
+using OxyPlot.Axes;
+
+namespace mzPlot
+{
+    public class BarPlot : Plot
+    {
+        /// <summary>
+        /// Creates a bar plot. The data X value is the height of the bar, and the data label is the X-axis label under the bar.
+        /// </summary>
+        public BarPlot(OxyPlot.Wpf.PlotView oxyPlotView, IEnumerable<Datum> data, OxyColor? borderColor = null, OxyColor? fillColor = null,
```

Comment:
is there an option for data labels? value? what about error bars?

Reply:
you can add this stuff by accessing the oxyplot model but it's kind of hard. I do want to add these features in a more intuitive way (probably adding xError, yError, zError in the Datum object) in a separate PR